spec/bolt/applicator_spec.rb

```diff
@@ -89,51 +89,6 @@
     )
   end
 
-  describe '#provide_puppet_missing_errors' do
```

Comment:
It looks like these tests were completely removed.

Reply:
oh I forgot to add the file they were moved to